src/exec.js

```diff
@@ -65,11 +65,14 @@ function execSync(cmd, opts) {
   var execCommand = '"'+process.execPath+'" '+scriptFile;
   var script;
 
+  opts.cwd = path.resolve(opts.cwd);
+  var optString = JSON.stringify(opts);
```

Comment:
This isn't safe against Unicode newlines (namely, U+2028 and U+2029): http://stackoverflow.com/a/9168133/1937836

Something like https://www.npmjs.com/package/js-stringify should fix this issue